changelog.d/10108.feature

```diff
@@ -0,0 +1 @@
+Implemented config option sso.update_display_name to keep user's display name in sync with the SSO display name.
```

Comment:
```suggestion
Implement config option `sso.update_display_name` to keep user's display name in sync with the SSO display name.
```

Just to keep in line with existing changelog entries, which are mostly in the present tense :slightly_smiling_face: